test/java/seedu/addressbook/commands/DeleteCommandTest.java

```diff
@@ -53,49 +55,61 @@ public void setUp() throws Exception {
 
     @Test
     public void execute_emptyAddressBook_returnsPersonNotFoundMessage() {
-        assertDeletionFailsDueToNoSuchPerson(1, emptyAddressBook, listWithEveryone);
+        assertDeletionFailsDueToNoSuchPerson(emptyAddressBook, listWithEveryone, 1);
+        assertDeletionFailsDueToNoSuchPerson(emptyAddressBook, listWithEveryone, new int[] {1, 2, 3, 4});
     }
 
     @Test
     public void execute_noPersonDisplayed_returnsInvalidIndexMessage() {
-        assertDeletionFailsDueToInvalidIndex(1, addressBook, emptyDisplayList);
+        assertDeletionFailsDueToInvalidIndex(addressBook, emptyDisplayList, 1);
+        assertDeletionFailsDueToInvalidIndex(addressBook, emptyDisplayList, new int[] {400, 1, 23});
     }
 
     @Test
     public void execute_targetPersonNotInAddressBook_returnsPersonNotFoundMessage()
             throws IllegalValueException {
-        Person notInAddressBookPerson = new Person(new Name("Not In Book"), new Phone("63331444", false),
+        Person notInAddressBookPerson1 = new Person(new Name("Not In Booka"), new Phone("63331444", false),
```

Comment:
Maybe can use the TestUtil class to generate a dummy person.

Reply:
Yes I could, thanks for the feedback!